src/scoreSheetBot.py

```diff
@@ -1112,6 +1112,22 @@ async def on_command_error(self, ctx: Context, error):
             traceback.print_exception(type(error), error, error.__traceback__, file=sys.stderr)
             lf.close()
 
+    ###########################################################################################
+    # Start test commands
+    # Test commands should be limited to dev only.
+    # No mutation should occur, and they are meant to test discord.py APIs.
+    ###########################################################################################
+
+    @commands.group(name='test')
+    async def test_group(self, ctx):
```

Comment:
Should add a decorator here making only certain roles able to run this command.

Reply:
Yes. Do you have one in mind? I would need the role to be able to use it. Or I can add a check to make sure env is in alpha

Reply:
Oh I forgot that I'm an admin in the test server so I'll look into it.

Reply:
You can just use 

`@role_call(STAFF_LIST)`